subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/AbstractCrossVersionGradleProfilerPerformanceTest.groovy

```diff
@@ -62,10 +62,11 @@ class AbstractCrossVersionGradleProfilerPerformanceTest extends Specification {
             new CsvGenerator(new File(debugArtifactsDirectory, "benchmark.csv")),
             new HtmlGenerator(new File(debugArtifactsDirectory, "benchmark.html"))
         )
+        def slackReporter = reporter
```

Comment:
What's the point of this?

Reply:
If the subsequent `Closure` is invoked from the subclass it does not have access to the private `reporter` field and will fail at runtime. I could also make the field protected to work around the problem. What do you prefer?

Reply:
I think it's fine, was just surprised about it.